backend/tally/settings.py

```
@@ -383,6 +383,11 @@ def get_required_env(key):
         'http://127.0.0.1:55010',
     ]))
 
+# Production logs only 5xx, so slow-but-successful requests were invisible.
+# Requests at or above this duration emit one sanitized WARNING. Tunable
+# without a deploy.
+SLOW_REQUEST_LOG_MS = int(os.environ.get('SLOW_REQUEST_LOG_MS', 1000))
```

Ruff (0.16.0)

[warning] Invalid type for environment variable default; expected `str` or `None`

(PLW1508)